README.md

```diff
@@ -15,7 +15,7 @@ The best source of information on running this software is the
 ### On the Amazon Linux AMI
 
 On the [Amazon Linux AMI](https://aws.amazon.com/amazon-linux-ami/), we provide an installable RPM which can be used via
-`sudo yum install ecs-init && sudo start ecs`. This is the recommended way to run it in this environment.
+`sudo yum install ecs-init && sudo systemctl start ecs`. This is the recommended way to run it in this environment.
```

Comment:
This section is specific to Amazon Linux (predecessor to Amazon Linux 2) which does not have `systemctl`. The command to start ECS Agent on Amazon Linux AMI is `sudo start ecs` which is correct in the README.

Reply:
Got it. I see that Amazon Linux 1 reached its end of life on December 31, 2023. Shouldn't we update the documentation for the current LTS versions of Amazon Linux? The end of life for Amazon Linux 1 was the reason that drove me to use Amazon Linux 2023, and ultimately, it's the rationale behind opening the Pull Request. Thank you for clarifying the status of Amazon Linux 1, which I was unsure about as well.

<img width="668" alt="image" src="https://github.com/aws/amazon-ecs-agent/assets/44049528/c8c7b843-d9dc-469e-8d79-e93af25bb798">